examples/webgl_gpu_picking_point.html

```diff
@@ -0,0 +1,291 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<title>three.js webgl - gpu - picking - point</title>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+	<style>
+		body {
+			color: #cccccc;
+			font-family: Monospace;
+			font-size: 13px;
+			text-align: center;
+
+			background-color: #050505;
+			margin: 0px;
+			overflow: hidden;
+		}
+
+		#info {
+			position: absolute;
+			top: 0px;
+			width: 100%;
+			padding: 5px;
+		}
+
+		a {
+			color: #0080ff;
+		}
+
+	</style>
+</head>
+<body>
+
+<div id="container"></div>
+<div id="info"><a href="http://threejs.org" target="_blank" rel="noopener">three.js</a> webgl - gpu - picking - point
+</div>
+
+<script type="module">
+/* 
+	Why use gpu to pick the object point?
+	1.Picking in cesium using gpu depth to calculate picking points，I think threejs can also have.
+	2.It is more efficient to calculate the picking point using gpu depth when the number of faces of a single model is large.
+	3.I can release the geometry data under the mesh, free more memory, and not affect the rendering.
+*/
+import * as THREE from '../build/three.module.js';
+import Stats from './jsm/libs/stats.module.js';
+import { GUI } from './jsm/libs/dat.gui.module.js';
+import { OrbitControls } from './jsm/controls/OrbitControls.js';
+
+
+var renderCamera, scene, renderer, stats, pickingScene, gui;
+
+var mesh;
+var mesh1;
+
+var controls;
+var raycaster;
+
+var pickingCamera;
+
+var GPUPickingPoint = new THREE.Vector3();
+var CPUPickingPoint = new THREE.Vector3();
+
+init();
+animate();
+
+function init() {
+
+	//
+
+	raycaster = new THREE.Raycaster();
+	renderCamera = new THREE.PerspectiveCamera( 60, window.innerWidth / window.innerHeight, 1, 10000 );
+	pickingCamera = new THREE.PerspectiveCamera( 60, window.innerWidth / window.innerHeight, 1, 10000 );
+	renderCamera.up.set( 0, 0, 1 );
+	pickingCamera.up.set( 0, 0, 1 );
```

Comment:
Why z-up in this demo?

Reply:
That's a habit caused by a model built by a colleague.